src/lib/errors.py

```diff
@@ -22,6 +22,10 @@ class ManifestLoadError(ManifestError):
     """Error loading flatpak-builder manifest"""
 
 
+class ManifestUnsupported(ManifestLoadError):
+    """Don't know how to handle manifest"""
```

Comment:
This exception is raised when a *source* is unsupported, not a whole manifest.

Reply:
This was intended as a generic exception when the input data is valid, but fedc doesn't know how to handle it. If limit it to sources, I suppose it should be renamed to e.g. `SourceUnsupported`?